sci-electronics/systemc/systemc-2.3.2.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+inherit eutils toolchain-funcs flag-o-matic
+MY_P="${P}"
+
+DESCRIPTION="A C++ based modeling platform for VLSI and system-level co-design"
+HOMEPAGE="http://accellera.org/community/systemc"
+SRC_URI="http://www.accellera.org/images/downloads/standards/${PN}/${MY_P}.tar.gz"
+
+SLOT="0"
+LICENSE="Apache-2.0"
+IUSE="doc static-libs cxx03 cxx11 +cxx14 cxx17"
+KEYWORDS="~amd64 ~x86"
+
+REQUIRED_USE="^^ ( cxx03 cxx11 cxx14 cxx17 )"
+
+RESTRICT="test"
+
+AUTOTOOLS_IN_SOURCE_BUILD=1
+
+S="${WORKDIR}/${MY_P}"
+
+src_configure() {
+	if use cxx17 ; then
```

Comment:
this will not fly. Use the sanest subset (imo C++14) and settle for that, anything else will drive me and QA insane.